src/pymodaq_plugins_thorlabs/hardware/kinesis.py

```diff
@@ -134,3 +136,47 @@ def get_position(self):
         else:
             position = 1
         return position
+
+
+class Piezo(Kinesis):
```

Comment:
I updated last week the kinesis module as I worked also on a new instrument from thorlabs: the benchtop DC Motors.

I added implemented a few more methods in the base class. Maybe they could be useful to you. Please do a fetch/sync from origin on your current branch. Make sure there is no issues and use whatever new functionnality is usefull to you (maybe none)!

Reply:
We fetch/sync from the origin branch and updated the hardware and daq_move to reflect the changes.

Reply:
hello are you ready for a review?

Reply:
Yes, I'm ready for a review.